app/views/payments/add.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :title, "Saldo opwaarderen voor #{@user.name}- SOFIA" %>
+
+<%= content_tag :div, id: 'payment-add', class: 'container footer-padding', data: {user: @user_json} do %>
```

Comment:
For the interactivity mentioned earlier today, the json should contain the current Saldo. Then, in Vue add a computed property `newSaldo` that return the sum of the value from the inputfield to the current saldo.